src/directives/decorators/stb-webmanual/add-row-button.html

```diff
@@ -0,0 +1,6 @@
+<div class="form-group"  ng-show="showCondition()">
+    <button class="stb-btn transparent light"
+            type="button"
+            name="addRowButton"
```

Comment:
we should write name="{{form.key}}". And again I think that button must be in separated task.